Benchmark/Benchmark/Form1.Designer.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Drawing;
+using System.Text;
+using System.Windows.Forms;
+
+namespace WindowsFormsApp2
+{
+    partial class Form1
+    {
+        /// <summary>
+        /// Required designer variable.
+        /// </summary>
+        private System.ComponentModel.IContainer components = null;
+
+        /// <summary>
+        /// Clean up any resources being used.
+        /// </summary>
+        /// <param name="disposing">true if managed resources should be disposed; otherwise, false.</param>
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing && (components != null))
+            {
+                components.Dispose();
+            }
+            base.Dispose(disposing);
+        }
+
+        #region Windows Form Designer generated code
+
+        /// <summary>
+        /// Required method for Designer support - do not modify
+        /// the contents of this method with the code editor.
+        /// </summary>
+        private void InitializeComponent()
+        {
+            this.components = new System.ComponentModel.Container();
+            this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font;
+            this.Text = "Form1";
```

Comment:
I'd suggest to rename it to "Simple Scalable Benchmark". It will help to distinguish the caption from other apps if some of them will not be killed by previous tests.

Reply:
Yep, good idea.

Reply:
Done.